dev-tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.9-bookworm
```

Comment:
is there a python version coincidentally called "bookwyrm" or am I misunderstanding this??

Reply:
Debian 12 is code-named "bookworm". :)

So these are the Python images based on Debian 12.

Reply:
wild!